collect_app/src/main/java/org/odk/collect/android/adapters/FileArrayAdapter.java

```diff
@@ -103,4 +105,22 @@ public View getView(int position, View convertView, ViewGroup parent) {
         }
         return v;
     }
+    @Override
+    public boolean isEnabled(int position) {
+        if (areAllItemsEnabled()) {
```

Comment:
Given that `areAllItemsEnabled()` is overridden to just return `enabled`, `isEnabled` is equivalent to `return enabled`. Please remove `areAllItemsEnabled()` and simplify the boolean return!